documentation/api/query/v4/operators.markdown

```diff
@@ -299,19 +299,59 @@ An `in` statement constitutes a full query string, which can be used alone or as
 
 "In" statements are **non-transitive** and take two arguments:
 
-* The first argument **must** consist of one or more **fields** for the endpoint or entity **being queried.**. This is a string or vector of strings.
+* The first argument **must** consist of one or more **fields** for the endpoint
+  or entity **being queried.**. This is a string or vector of strings.
 * The second argument **must** be either:
-** an **`extract` statement,** which acts as a list of fields to extract during the subquery for matching against the **fields** in the `in` clause.
-** a **`from` statement,** which sets the context, and allows for an extract statement to be provided. *Note:* this syntax is new and experimental.
+** an **`extract` statement,** which acts as a list of fields to extract during
+   the subquery for matching against the **fields** in the `in` clause.
+** a **`from` statement,** which sets the context, and allows for an extract
+   statement to be provided. *Note:* this syntax is new and experimental.
+** an **`array` statement,** which acts as a list of values to match against the
+   **field** in the `in` clause.
 
 **Matches if:** the field values are included in the list of values created by the `extract` or `from` statement.
 
+##### `array`
+
+An `in` statement also accepts an `array` statement as a second argument.
+
+"Array" statements take a single vector argument of values to match the first
+argument of `in` against.
+
+The following query filters for the nodes, `foo.local`, `bar.local`, and
+`baz.local`:
+
+    ["in", "certname",
+     ["array",
```

Comment:
I think a literal decorator is fine, if its required to remove ambiguity. We'll want to use the syntax elsewhere though, if we need to provide a literal - the main case I can think of is for equality TBH. Anyway - just wanted to ponder it, I think this is fine.